server/repository/src/mock/full_master_list.rs

```diff
@@ -145,3 +146,34 @@ pub fn mock_master_list_master_list_line_filter_test() -> FullMockMasterList {
         ],
     }
 }
+
+pub fn mock_master_list_default_price_list() -> FullMockMasterList {
```

Comment:
I checked, looks like we (I) just created a master list in the test case for item pricing (default_price_list).
This test case doesn't include master lists just `MockDataInserts::none().items().names().stores(),`

I think we have undefined behaviour if we have more than one `is_default_price_list` right? It will just pick the first one it finds.

All that is to say I think we're ok to add this but wanted to think about implications.
Maybe should even have a comment here saying there should only ever be one?